sdk/eventhubs/ci.yml

```diff
@@ -30,7 +30,7 @@ stages:
       CtestRegex: "azure-messaging-eventhubs.*"
       LiveTestCtestRegex: "azure-messaging-eventhubs.*"
       LiveTestTimeoutInMinutes: 120
-      LineCoverageTarget: 22
+      LineCoverageTarget: 20
```

Comment:
:(

Reply:
Believe me, I know - this is NOT pretty.